athena_cfn/athena-bucket-cfn.yml

```diff
@@ -0,0 +1,233 @@
+# Purpose:  Expose Metadata from a single Quilt bucket in Athena
+#===============================================================================
+# https://aws.amazon.com/cloudformation/resources/templates/
+# https://catalog.us-east-1.prod.workshops.aws/workshops/9981f1a1-abdc-49b5-8387-cb01d238bb78/en-US/20-howtostart/201-self-paced/2013-cloudformation
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/walkthrough-crossstackref.html
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Configure Athena tables for a Quilt Bucket
+Parameters:
+  QuiltAthenaStack:
```

Comment:
I'm confused by the word stack here. The whole thing is a stack? So unless there's a child template, I wouldn't use that word for a param. I think this is QuiltAthenaPrefix and explain that it's an identifer for db, workgroup, bucket, similar to what you do below.

Reply:
Is this actually WorkgroupName?

Reply:
This is a link back to the **previous** stack, created by athena-cfn.  This bucket stack needs to be run multiple times, and reference the parent stack. Is there any easier way to do that?